static/monsterSort.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	$('select').change(function() {
+		selection = $(this).val();
+		if (selection == 'all') {
+			$('.monster-box').show();
+		} else {
+			var selector = '.' + selection;
```

Comment:
Might want to mention somewhere in a comment that the values are the same as class names on the monster boxes.